### PR TITLE
backup local policies to history when deleting it

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>
Refers: https://issues.redhat.com/browse/ACM-5703
Blocked By: https://github.com/stolostron/multicluster-global-hub/pull/454

### CODE REVIEW

File: operator/pkg/controllers/hubofhubs/database/3.tables.sql [hunk 1]
```diff
@@ -1,83 +1,122 @@
...
 CREATE TABLE IF NOT EXISTS  history.subscriptions (
-    id uuid NOT NULL,
+    id uuid PRIMARY KEY,
     payload jsonb NOT NULL,
     created_at timestamp without time zone DEFAULT now() NOT NULL,
     updated_at timestamp without time zone DEFAULT now() NOT NULL,
     deleted boolean DEFAULT false NOT NULL
 );
 
+CREATE TABLE IF NOT EXISTS history.local_compliance (
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/database/3.tables.sql:81]: It should be committed in the previous PR.

File: operator/pkg/controllers/hubofhubs/database/3.tables.sql [hunk 2]
```diff
@@ -293,90 +313,6 @@ CREATE TABLE IF NOT EXISTS event.local_policies (
     CONSTRAINT local_policies_unique_constraint UNIQUE (policy_id, cluster_id, created_at)
 );
 
-ALTER TABLE history.applications DROP CONSTRAINT IF EXISTS applications_pkey;
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/database/3.tables.sql:296]: why remove those?

File: operator/pkg/controllers/hubofhubs/database/3.tables.sql (hunk 1 above)
Comment (yanmxa) [reply operator/pkg/controllers/hubofhubs/database/3.tables.sql:81]: Yes, after the previous PR merge. This will disappear.

**Pull request merged**